primefaces/src/main/resources/META-INF/resources/primefaces/menu/menu.base.js

```diff
@@ -136,19 +136,20 @@ PrimeFaces.widget.Menu = PrimeFaces.widget.BaseWidget.extend({
      */
     bindPanelEvents: function() {
         var $this = this;
+        var $menu = this.menu || this.jq;
```

Comment:
i dont like this TBH, its a bit dirty
we should add a method like getMenuElement and overwrite it

Reply:
ok got it!

Reply:
Refactored to a `getMenu()` method that is overridden in MenuButton.